rpc/src/parsed_token_accounts.rs

```diff
@@ -75,7 +77,10 @@ where
 /// program_id) and decimals
 pub fn get_mint_owner_and_decimals(bank: &Arc<Bank>, mint: &Pubkey) -> Result<(Pubkey, u8)> {
     if mint == &spl_token_native_mint() {
-        Ok((spl_token_id(), spl_token::native_mint::DECIMALS))
+        Ok((
+            spl_token_native_mint_program_id(),
```

Comment:
This is an interesting case to think about: would each SPL Token instance have a native mint, or only `Tokenkeg...`. 

If each instance I think we need a new token instruction to initialize the native mint, since for `Tokenkeg...` we ultimately did this early on with a bank.rs adjustment

Reply:
Yes, I think we will need a native mint for each program.

Pity we didn't put the native mint at a derived address; then it would be a bit easier to find the native mint for each token program.

I'll think on this, and initialization, etc. Perhaps we can fix up the native mint for the new token program to finally adjust the supply correctly on wrap and unwrap.